{"name": "test_first_heading", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestSignUp' object has no attribute 'driver'", "trace": "self = <signup_test.TestSignUp object at 0x000001FAE0AE7910>\n\n    @pytest.fixture(scope='function', autouse=True)\n    def signup_fixture(self):\n>       self.sign = SignUp_Page(self.driver)\nE       AttributeError: 'TestSignUp' object has no attribute 'driver'\n\ntests\\signup_test.py:18: AttributeError"}, "description": "Check the SignUp Page", "start": 1730126547792, "stop": 1730126547792, "uuid": "c2dfff3e-accc-48f4-9542-a9b5ce88e891", "historyId": "b41f8944632e339e7c42bf1f30eb0818", "testCaseId": "b41f8944632e339e7c42bf1f30eb0818", "fullName": "tests.signup_test.TestSignUp#test_first_heading", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "signup_test"}, {"name": "subSuite", "value": "TestSignUp"}, {"name": "host", "value": "MSI"}, {"name": "thread", "value": "29152-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.signup_test"}]}